minimal.cmake

```diff
@@ -0,0 +1,16 @@
+hunter_config(
+    xgboost
+    VERSION 0.40-p10
+    CMAKE_ARGS
+    XGBOOST_USE_HALF=ON
+    XGBOOST_USE_CEREAL=ON
+    XGBOOST_DO_LEAN=ON
+)
+
```

Comment:
```
# This will be shared by two modules, so we introduce two matching options:
option(DRISHTI_BUILD_OGLES_GPGPU "Build ogles_gpgpu" ON)
option(ACF_BUILD_OGLES_GPGPU "Build ogles_gpgpu" ON)
if(DRISHTI_BUILD_OGLES_GPGPU OR ACF_BUILD_OGLES_GPGPU)
    set(HUNTER_ACF_BUILD_OGLES_GPGPU ON)
else()
    set(HUNTER_ACF_BUILD_OGLES_GPGPU OFF)
endif()

hunter_config(
     acf
     VERSION
     ${HUNTER_acf_VERSION}
     CMAKE_ARGS
     ACF_BUILD_OGLES_GPGPU=${HUNTER_ACF_BUILD_OGLES_GPGPU}
 )
```

:point_up: ?

Reply:
There is no need to have `option` inside `config.cmake`, they can be declared in CMakeLists.txt. Just need to do it before first `hunter_add_package` command.

Reply:
> HUNTER_ACF_BUILD_OGLES_GPGPU

`HUNTER_*` is for Hunter related options, since variable is local and used only inside one file we can name it simply `afc_build_ogles_gpgpu`.

Reply:
Also the goal of "minimal" config is to be minial :) `ACF_BUILD_OGLES_GPGPU=ON` was added because `ACF_BUILD_OGLES_GPGPU=OFF` is not working. Probably `option`s should not be used here at all.

Reply:
> Also the goal of "minimal" config is to be minimal 

Okay